crates/sui/src/client_commands.rs

```diff
@@ -1786,6 +1767,52 @@ pub struct NewAddressOutput {
     pub recovery_phrase: String,
 }
 
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ObjectsOutput {
+    pub object_id: ObjectID,
+    pub version: SequenceNumber,
+    pub digest: String,
+    pub owner_type: String,
```

Comment:
Note that if this is only used to display output from `sui client objects`, then you don't need to provide an `owner_type` field -- it will always be `AddressOwner`.